playbooks/csi_driver_config.yaml

```diff
@@ -0,0 +1,5 @@
+---
+- name: CSI Driver configuration
+  hosts: bastion
```

Comment:
Did you consider HA scenarios here with 2 bastion node?

Reply:
Fixed, running on one bastion should be fine.